universal-login-relayer/lib/http/relayers/Relayer.ts

```diff
@@ -98,11 +99,17 @@ class Relayer {
 
   runServer() {
     this.app = express();
+    this.app.set('trust proxy', true);
```

Comment:
Why?

Reply:
so the `req.protocol` is assigned from `X-Forwarded-Proto`